com.github.gkarsay.parlatype.json

```diff
@@ -0,0 +1,58 @@
+{
+    "app-id": "com.github.gkarsay.parlatype",
+
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.24",
+    "sdk": "org.gnome.Sdk",
+
+    "command": "parlatype",
+
+    "tags": [],
+
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+
+        "--filesystem=host",
```

Comment:
Haven't tried the application yet but it seems like it only opens a single file? Shouldn't need host access right?

Reply:
Removed in new commit. I wasn't aware that the file chooser doesn't need it for opening files. Parlatype actually also tries to write some metadata (last position) to the file with `g_file_set_attributes_from_info()` but it didn't work with `--filesystem=host`.